tests/core/serviceinstance_test.py

```diff
@@ -7,11 +7,9 @@
 from testify import setup
 from testify import setup_teardown
 from testify import TestCase
-from testify.assertions import assert_in
 from testify.assertions import assert_not_equal
 
 from tests.assertions import assert_length
-from tests.testingutils import autospec_method
```

Comment:
Since you're not getting rid of autospec_method, maybe we can bring it back in this file?

Reply:
we will gradually replace it with create_autospec. i'm genuinely afraid of that thing considering that googling for it returns this repository in the first search result: https://www.google.com/search?q=python+autospec_method&nfpr=1